### PR TITLE
Hard fork `ClaimStakeReward` from 1 to 2

### PR DESCRIPTION
**Pull request by boscohyun**
This pull request based on #1370 

Resolve https://github.com/planetarium/NineChronicles/issues/1933

### CODE REVIEW

**Review by moreal — CHANGES_REQUESTED** [review]
If there is a chance to restart the chain, we must consider hard a strategy for actions and models versioning

File: .Lib9c.Tests/Action/ClaimStakeRewardTest.cs [hunk 1]
```diff
@@ -99,14 +137,5 @@ public void Execute()
...
         }
-
-        [Fact]
-        public void Serialization()
-        {
-            var action = new ClaimStakeReward(_avatarAddress);
-            var deserialized = new ClaimStakeReward();
-            deserialized.LoadPlainValue(action.PlainValue);
-            Assert.Equal(action.AvatarAddress, deserialized.AvatarAddress);
-        }
```
[the remark this answers is not in the document]
Comment (boscohyun) [reply .Lib9c.Tests/Action/ClaimStakeRewardTest.cs:110]: @moreal You can check this test at line number 100. That's not removed.😅

[the remark this answers is not in the document]
Comment (moreal) [reply .Lib9c.Tests/Action/ClaimStakeRewardTest.cs:110]: It's too hard to look the changes 😞 I see, thanks!

File: Lib9c/Action/ClaimStakeReward.cs [hunk 2]
```diff
@@ -11,26 +10,62 @@
 
 namespace Nekoyume.Action
 {
-    [ActionType("claim_stake_reward")]
+    /// <summary>
+    /// Hard forked at https://github.com/planetarium/lib9c/pull/1371
+    /// </summary>
+    [ActionType("ActionTypeText")]
```
[the remark this answers is not in the document]
Comment (boscohyun) [reply Lib9c/Action/ClaimStakeReward.cs]: @moreal Fixed! thanks!

[the remark this answers is not in the document]
Comment (moreal) [reply Lib9c/Action/ClaimStakeReward.cs]: @boscohyun If you have no antipathy for force-push, could you amend the `Fix typo` commit into `Hard fork ClaimStakeReward from 1 to 2`. I'm worrying the case when someone cherry-pick the `Hard fork ClaimStakeReward from 1 to 2` commit only.

File: .Lib9c.Tests/Action/ClaimStakeRewardTest.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (boscohyun) [reply .Lib9c.Tests/Action/ClaimStakeRewardTest.cs:110]: Sorry for moving some features. For encapsulation, I had to change the access modifiers of some functions to private. The code analyzer then drew a red line, guiding me to change the order of the functions to public-first.😇

File: Lib9c/Action/ClaimStakeReward.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (boscohyun) [reply Lib9c/Action/ClaimStakeReward.cs]: @moreal Oh, I cannot think about that case. Thanks for your review.

File: .Lib9c.Tests/Action/ClaimStakeRewardTest.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (moreal) [reply .Lib9c.Tests/Action/ClaimStakeRewardTest.cs:110]: You're right, I don't want to blame you. It's just my mistake! 👍🏻

File: Lib9c/Action/ClaimStakeReward.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (boscohyun) [reply Lib9c/Action/ClaimStakeReward.cs]: @moreal Applied!

[the remark this answers is not in the document]
Comment (moreal) [reply Lib9c/Action/ClaimStakeReward.cs]: After the based pull request was merged, this pull request may be able to merge.

[the remark this answers is not in the document]
Comment (boscohyun) [reply Lib9c/Action/ClaimStakeReward.cs]: That's exactly what I want.

**Review by ipdae — APPROVED** [review]

File: Lib9c/Action/ClaimStakeReward.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (boscohyun) [reply Lib9c/Action/ClaimStakeReward.cs]: @moreal The based pull request was merged!

File: Lib9c/Action/AccountStateDeltaExtensions.cs [hunk 3]
```diff
@@ -872,7 +872,10 @@ public static ShopState GetShopState(this IAccountStateDelta states)
             return (arenaInfoAddress, arenaInfo, isNew);
         }
 
-        public static bool TryGetStakeState(this IAccountStateDelta states, Address agentAddress, out StakeState stakeState)
+        public static bool TryGetStakeState(
+            this IAccountStateDelta states,
+            Address agentAddress,
+            out StakeState stakeState)
```
Comment (moreal) [inline Lib9c/Action/AccountStateDeltaExtensions.cs:878]: It's minor but I hope the part formatting code separated as other pull request or commit.

**Review by moreal — APPROVED** [review]

**Pull request merged**